### ISSUE TITLE
✅ 🐞 Bug Report: 3 Warnings on new Theme Builder when Landing Page is disabled when using PHP 8.2 - [ED-11843]

### ISSUE THREAD

**Issue by banlieues-asbl** [elementor/elementor#22991]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

### Description

When you disable Landing page (in Elementor features) and then go to the new theme builder page, when the theme builder page loads, 3 warning messages appear with Elementor Pro + PHP 8.2.6.

Warning: Attempt to read property "labels" on null in D:\xampp\htdocs\wordpress\themes\wp-content\plugins\elementor\app\modules\import-export\module.php on line 733

Warning: Attempt to read property "singular_name" on null in D:\xampp\htdocs\wordpress\themes\wp-content\plugins\elementor\app\modules\import-export\module.php on line 733

Warning: Attempt to read property "label" on null in D:\xampp\htdocs\wordpress\themes\wp-content\plugins\elementor\app\modules\import-export\module.php on line 734

### Steps to reproduce

1. Disable landing page
2. Go to New theme builder

### Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [x] I can reproduce this bug consistently following the steps above.

### System Info

Win 10 + PHP 8.2.6

**Comment by nicholaszein** [comment]
Hello @banlieues-asbl , 

Thank you for submitting your report! 🙏 

### ❌ While reviewing your report, we noticed it was missing some key pieces of information:

- 🔬 **Isolate the problem:** Deactivate all of your plugins (except Elementor/Pro), and temporarily change your WordPress theme to [Hello theme](https://wordpress.org/themes/hello-elementor/). If this solves the problem, gradually activate your plugins and theme one by one, until you spot the conflicting plugin or theme.

- 🕵️ **Validate Your Issue:** If the issue can't be recreated consistently or reproduced on other sites, it's likely not an Elementor bug. In that case, your issue is likely caused by a conflict with something else.

- 💾 **System info:** Please include the System Info report as outlined in our [guidelines](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml). 
   - Get your System Info from your WordPress `Dashboard` > `Elementor` > `System info` [learn how to copy it here](https://elementor.com/help/hare-my-system-info/). We can't accept issues without it.
   - 
Without them, we simply can't help. 😕 

### 🧐 Please provide the missing information above so we can troubleshoot your issue faster and better.

Best regards

**Comment by banlieues-asbl** [comment]
[system-info-localhost-11-07-2023.txt](https://github.com/elementor/elementor/files/12012594/system-info-localhost-11-07-2023.txt)

![Capture22](https://github.com/elementor/elementor/assets/84187112/4231003e-b25c-442a-ac82-acfeb7f01611)

1. Occurs with all plugins disabled
2. Occurs with hello elementor (or hello child) theme
3. Happens on all my wp + elementor sites

**Comment by jamieburchell** [comment]
Me too. After clicking "Theme Builder" I get log spam

```
[25-Jul-2023 10:33:54 UTC] PHP Warning:  Attempt to read property "label" on null in /var/www/web/app/plugins/elementor/app/modules/import-export/module.php on line 734
[25-Jul-2023 10:33:55 UTC] PHP Warning:  Undefined array key "data" in /var/www/web/app/plugins/elementor/core/common/modules/ajax/module.php on line 175
[25-Jul-2023 10:34:51 UTC] PHP Warning:  Attempt to read property "labels" on null in /var/www/web/app/plugins/elementor/app/modules/import-export/module.php on line 733
[25-Jul-2023 10:34:51 UTC] PHP Warning:  Attempt to read property "singular_name" on null in /var/www/web/app/plugins/elementor/app/modules/import-export/module.php on line 7
```

**Comment by sekhsoyebali** [comment]
Please fix this issue asap

**Comment by nicholaszein** [comment]
Hey there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.17! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
